src/style/design-tokens/theme-compatibility.util.spec.js

```diff
@@ -0,0 +1,36 @@
+import themeCompatibility from "./theme-compatibility.util";
+import mintTheme from "../themes/mint";
+
+describe("themeCompatibility", () => {
+  it("should create clone given theme before extending it", () => {
+    const output = themeCompatibility(mintTheme);
```

Comment:
could we move this `const` upper block under the `describe` - this will allow us to delete lines 12 and 20

Reply:
eventually this whole utility is gone.